server/SwipeDecision.py

```diff
@@ -15,10 +16,58 @@ def getPotentialMatchList(currentUserId):
             cursor.execute(PotentialMatchQuery, userID)
             potentialListId = [i[0] for i in cursor.fetchall()]
```

Comment:
maybe when you execute the potential match query retrieve the user id and the locations so you don't need to call the database gain later